### PR TITLE
Add nomination submission form

### PR DESCRIPTION
**Pull request by JakeHartnell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,11 +16,13 @@
...
+### Submission / Scoring Process
 
-- Community members will make on chain proposals nominating people for awards.
-- For nominations that pass, judges will determine the final score based on usefulness of the contribution to the Juno ecosystem.
-- Funds will be allocated according to points.
+- [Fill out the Nominiation form](https://docs.google.com/forms/d/1doYtar6ldlnFKedbeZtRka4g9j3aO1Gb_MaWxllHw7E/edit) (for yourself or someone else).
+- Judges will determine the final score based on usefulness of the contribution to the Juno ecosystem.
+- Rewards will be allocated according to score.
+
+Judges will include the Juno Development Commity and some notable individuals from the Cosmos Ecosystem (to be announced). Judges are not eligible to receive Hack Juno rewards.
```
Comment (blockcreators) [inline README.md:25]: I think you might have meant committee?

**Review by blockcreators — CHANGES_REQUESTED** [review]
Typo?

File: README.md (hunk 1 above)
Comment (JakeHartnell) [reply README.md]: Indeed, fixed.

**Review by blockcreators — APPROVED** [review]
looks good

**Pull request merged**